.gitattributes

```diff
@@ -0,0 +1,9 @@
+# Normalize line endings to auto.
+* text auto
+
+# Ensure the following are treated as binary.
+*.jar      binary
```

Comment:
Also, I think it would be good to locally run:

```
git rm --cached -r .
git add -A
```

What this will do is show you any files in the repo that `.gitattributes` would apply itself to. I think the files from this output are candidates for being treated as binary. When I ran these myself, `png` was the one that caught my attention, but there may be others.

Reply:
Thanks @jzheaux , I did that and added `png` to the list. I think the other files that showed up are plain text